homeassistant/components/openuv/config_flow.py

```diff
@@ -54,16 +42,13 @@ async def async_step_import(self, import_config):
 
     async def async_step_user(self, user_input=None):
         """Handle the start of the config flow."""
-
         if not user_input:
             return await self._show_form()
 
-        identifier = (
-            f"{user_input.get(CONF_LATITUDE, self.hass.config.latitude)}, "
-            f"{user_input.get(CONF_LONGITUDE, self.hass.config.longitude)}"
-        )
-        if identifier in configured_instances(self.hass):
-            return await self._show_form({CONF_LATITUDE: "identifier_exists"})
+        identifier = f"{user_input[CONF_LATITUDE]}, {user_input[CONF_LONGITUDE]}"
```

Comment:
Maybe do a special identifier if latitude/longitude are not given instead of rendering it as `None, None`